src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -235,7 +235,7 @@ public async Task NoCallback_RevokedCertificate_NoRevocationChecking_Succeeds()
             }
         }
 
-        [OuterLoop("Uses external server")]
+        [OuterLoop("Uses external servers")]
```

Comment:
Do we really need this change? It makes the diff really hard to read.

Reply:
It was suggested by Mitchell we should unify it and I agree. I separated it into different commit on this PR to make it bit easier to review.